models/quality_measures/intermediate/nqf_0097_medication_reconciliation_post_discharge/quality_measures__int_nqf0097_numerator.sql

```diff
@@ -0,0 +1,101 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with denominator as (
+
+    select
+          patient_id
+        , performance_period_begin
+        , performance_period_end
+        , measure_id
+        , measure_name
+        , measure_version
+    from {{ ref('quality_measures__int_nqf0097_denominator') }}
+
+)
+
+, reconsilation_codes as (
+
+    select
+          concept_name
+        , code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) = 'medication reconciliation post discharge'
```

Comment:
Please include concept `Encounter to Document Medications` and related core table to cater snomed codes as well.